web/toggle_button.css

```
@@ -0,0 +1,219 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.toggle-button {
+  --button-background-color: #f0f0f4;
+  --button-background-color-hover: #e0e0e6;
+  --button-background-color-active: #cfcfd8;
+  --color-accent-primary: #0060df;
+  --color-accent-primary-hover: #0250bb;
+  --color-accent-primary-active: #054096;
+  --border-interactive-color: #8f8f9d;
+  --border-radius-circle: 9999px;
+  --border-width: 1px;
+  --size-item-small: 16px;
+  --size-item-large: 32px;
+  --color-canvas: white;
+}
+
+@media (prefers-color-scheme: dark) {
+  .toggle-button {
```

Can't we just define this as follows instead, directly under `.toggle-button`, which is a tiny bit more compact?
```suggestion

  @media (prefers-color-scheme: dark) {
```